queue_services/business-bn/tests/unit/bn_processors/test_admin.py

```diff
@@ -0,0 +1,162 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Test Suites to ensure that the admin is operating correctly."""
+import uuid
+import xml.etree.ElementTree as Et
+
+import pytest
+from business_model.models import Business, RequestTracker
+
+from business_bn.resources.business_bn import process_event
+from tests.unit import create_registration_data
+
+
+acknowledgement_response = """<?xml version="1.0"?>
+    <SBNAcknowledgement>
+        <header>
+            <transactionID>BNTUQ0E1OC0</transactionID>
+        </header>
+        <body>A Valid Document Type was received.</body>
+    </SBNAcknowledgement>"""
+
+
+@pytest.mark.parametrize('request_type,business_number', [
+    ('BN15', '993775204'),
+    ('BN15', ''),
+    ('BN15', None),
+    ('RESUBMIT_INFORM_CRA', None),
+])
+@pytest.mark.asyncio
```

Comment:
this and others in the tests should be removed as you aren't using asyncio anymore